evalml/objectives/standard_metrics.py

```diff
@@ -345,6 +345,17 @@ def objective_function(self, y_true, y_predicted, X=None):
         return metrics.mean_absolute_error(y_true, y_predicted)
 
 
+class MAPE(RegressionObjective):
+    """Mean absolute percentage error for time series regression."""
+    name = "Mean Absolute Percentage Error"
+    greater_is_better = False
+    score_needs_proba = False
+    perfect_score = 0.0
+
+    def objective_function(self, y_true, y_predicted, X=None):
```

Comment:
Maybe we should just raise a ValueError if any `y_true` are 0? My concern is that 0 can be a meaningful target value (like stock returns) and users may be mislead that those records are being used in the evaluation when they are not.